insights/parsers/satellite_postgresql_query.py

```diff
@@ -72,15 +72,24 @@ class SatellitePostgreSQLQuery(CommandParser, list):
 
     Raises:
         SkipException: when there isn't data in the table
-        ParseException: when the output isn't in good csv format
+        ParseException: when the output isn't in good csv format or the subclass doesn't define headers
     """
+    headers = []
 
     def parse_content(self, content):
-        if not content or len(content) == 1:
-            raise SkipException("There is no data in the table")
+        if self.__class__.__name__ != 'SatellitePostgreSQLQuery' and not self.headers:
+            raise ParseException("Please define the headers attribute which contains the column names.")
+        start_index = None
```

Comment:
the `insights.parsers.calc_offset` can be used to get the `start_index`.

Reply:
Updated, please help to check again, thanks.